hack/create-kind-cluster.sh

```diff
@@ -22,11 +22,25 @@ set -o pipefail
 CLUSTER_SUFFIX=${CLUSTER_SUFFIX:-"cluster.local"}
 NODE_VERSION=${NODE_VERSION:-"v1.23.3"}
 NODE_SHA=${NODE_SHA:-"sha256:0df8215895129c0d3221cda19847d1296c4f29ec93487339149333bd9d899e5a"}
+REGISTRY_NAME=${REGISTRY_NAME:-"kind-registry"}
+REGISTRY_PORT=${REGISTRY_PORT:-"5001"}
+
+# create registry container unless it already exists
+if [ "$(docker inspect -f '{{.State.Running}}' "${REGISTRY_NAME}" 2>/dev/null || true)" != 'true' ]; then
+  docker run \
+    -d --restart=always -p "127.0.0.1:${REGISTRY_PORT}:5000" --name "${REGISTRY_NAME}" \
+    docker.io/registry:2
+fi
```

Comment:
Similar to my question above, I didn't get the reason for this

Reply:
See https://github.com/knative-sandbox/eventing-kafka-broker/pull/1968#discussion_r822713028